packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/get.js

```diff
@@ -314,6 +314,17 @@ describe('plugin-conversation', function () {
           assert.equal(conversations[0].displayName, conversation1.displayName);
         }));
 
+      describe('with summary = true', () => {
+        it('retrieves a set of conversations', () => webex.internal.conversation.list({
+          summary: true,
+          conversationsLimit: 2
+        })
+          .then((conversations) => {
+            assert.include(map(conversations, 'url'), conversation1.url);
+            assert.include(map(conversations, 'url'), conversation2.url);
+          }));
```

Comment:
Can you assert that the number of conversations requested is the number returned?